src/components/checkbox/index.tsx

```diff
@@ -241,14 +261,21 @@ class Checkbox extends Component<CheckboxProps, CheckboxState> {
     return label ? (
       <View row centerV style={containerStyle}>
         {this.renderCheckbox()}
-        <Text flexS style={[this.styles.checkboxLabel, labelStyle]} recorderTag={'unmask'} {...labelProps} onPress={this.onPress}>
+        <Text flexS style={[this.styles.checkboxLabel, this.getLabelStyle(), labelStyle]} recorderTag={'unmask'} {...labelProps} onPress={this.onPress}>
```

Comment:
I'd move the `this.styles.checkboxLabel` (with the array) to the function and memoize it.

Reply:
Non of the styles on this component are memorized. We should take it to a different task